src/librbd/io/SimpleSchedulerObjectDispatch.h

```diff
@@ -17,6 +17,8 @@ class ImageCtx;
 
 namespace io {
 
+class LatencyStats;
```

Comment:
Nit: perhaps forward declare as an inner class of the the dispatch class?

Reply:
I had some issues when linking mock tests due to the templatized dispatch class, but I did not try hard, just moved it outside. Ok I will try again.

Reply:
If it's a pain, no worries.